src/app/route/ProductPage/ProductPage.component.js

```diff
@@ -112,14 +114,24 @@ export class ProductPage extends PureComponent {
         return (
             <>
                 { this.renderCustomizableOptions() }
-                <ProductInformation
-                  product={ { ...dataSource, parameters } }
-                  areDetailsLoaded={ areDetailsLoaded }
-                />
-                <ProductReviews
-                  product={ dataSource }
-                  areDetailsLoaded={ areDetailsLoaded }
-                />
+                <ProductTabs>
+                    <ProductInformation
+                      product={ { ...dataSource, parameters } }
+                      areDetailsLoaded={ areDetailsLoaded }
+                      tabName={ __('About') }
+                    />
+                    <ProductInformationAttributes
+                      product={ { ...dataSource, parameters } }
+                      areDetailsLoaded={ areDetailsLoaded }
+                      tabName={ __('Details') }
+                    />
+                    <ProductReviews
+                      product={ dataSource }
+                      areDetailsLoaded={ areDetailsLoaded }
+                      tabName={ __('Reviews') }
+                    />
```

Comment:
Great API! I like it very much. Nonetheless, I did not see any prop validation inside of the ProductTabs - you should throw an error there if a `tabName` is not supplied.